pkg/resource/scalable_target/manager_test_suite_test.go

```diff
@@ -17,38 +17,38 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/aws-controllers-k8s/applicationautoscaling-controller/pkg/testutil"
 	mocksvcsdkapi "github.com/aws-controllers-k8s/applicationautoscaling-controller/test/mocks/aws-sdk-go/applicationautoscaling"
 	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
-	"github.com/ghodss/yaml"
-	"github.com/aws-controllers-k8s/applicationautoscaling-controller/pkg/testutil"
+	ackmetrics "github.com/aws-controllers-k8s/runtime/pkg/metrics"
 	acktypes "github.com/aws-controllers-k8s/runtime/pkg/types"
 	svcsdk "github.com/aws/aws-sdk-go/service/applicationautoscaling"
+	"github.com/ghodss/yaml"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"go.uber.org/zap/zapcore"
 	"path/filepath"
-	"testing"
 	ctrlrtzap "sigs.k8s.io/controller-runtime/pkg/log/zap"
-	ackmetrics "github.com/aws-controllers-k8s/runtime/pkg/metrics"
-	"go.uber.org/zap/zapcore"
+	"testing"
```

Comment:
Needs `goimports` with probably some manual reorganization

Reply:
Amine, we will address this in a separate PR.